### PR TITLE
Hw06 email

### PR DESCRIPTION
**Pull request by OIgaB**

### CODE REVIEW

File: helpers/sendEmail.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+const { SENDGRID_API_KEY, EMAIL_FROM } = process.env; // забираємо зі змінних оточень ключ до sendgrid
+
+sgMail.setApiKey(SENDGRID_API_KEY); // передаємо sendgrid ключ
+
+// const email = {
+//     to: 'fitacaw793@royalka.com',
+//     from: 'своє мило',
+//     subject: 'Test email',
+//     html: '<p><strong>Test email</strong> from localhost:3000</p>'
```
Comment (ceREAL133) [inline helpers/sendEmail.js]: delete comments

File: controllers/auth.js [hunk 2]
```diff
@@ -32,12 +42,50 @@ const register = async(req, res) => {
...
+    const { email } = req.body;
+    const user = await User.findOne({ email });
+    if(!user){
+        throw HttpError(401, 'Email is not found');
+    }
+    if(user.verify) { //якщо користувач вже підтвердив email (user.verify = true)
+        throw HttpError(401, 'The email is already verified')   // не спрацює, бо спочатку Postman видасть "Email already in use"
+    }
+    const verifyEmail = {
+        to: email,
```
Comment (ceREAL133) [inline controllers/auth.js]: you have already used this piece of code. Reuse it

**Review by ceREAL133 — APPROVED** [review]

File: helpers/sendEmail.js (hunk 1 above)
Comment (OIgaB) [reply helpers/sendEmail.js]: Done!

File: controllers/auth.js (hunk 2 above)
Comment (OIgaB) [reply controllers/auth.js]: Done! In 'helpers' directory created the file 'createVerificationEmail.js' with function that was reused in 'register' and 'resendVerifyEmail' functions.

**Pull request merged**